klippy/extras/ds18b20.py

```diff
@@ -10,6 +10,8 @@
 # Temperature can be sampled at any time but conversion time is ~750ms, so
 # setting the time too low will not make the reports come faster.
 DS18_MIN_REPORT_TIME = 1.0
+DS18_MAX_CONSECUTIVE_ERRORS = 4
```

Comment:
Thanks for reviewing.

> I'm not that familiar with Python coding style in this project or generally so I can't say if they're worth keeping.

It can be convenient to put code constants in the Python code, as it's faster to test and debug changes that way.  (One only need change the python file and restart, instead of flashing a micro-controller.)  I don't have a particular preference for this code constant.

> I think the rest of the changes to this file are all formatting changes.

I agree that functional code changes and formatting/whitespace code changes should not be mixed in the same commit.  There's a note on this at https://www.klipper3d.org/CONTRIBUTING.html#what-to-expect-in-a-review (point 6).

-Kevin

Reply:
I have force-pushed new commits which separate the formatting changes.